src/lib/http.coffee

```diff
@@ -21,11 +23,30 @@ module.exports = (dnschain) ->
             @log = gNewLogger 'HTTP'
             @log.debug "Loading HTTPServer..."
             @rateLimiting = gConf.get 'rateLimiting:http'
+            app = express()
+            router = express.Router()
+
+            router.route(/\/v1\/(\w+)\/(\w+)(\/([^\/]+)(\/(\w+)(\.(\w+))?)?)?/)
+            .get (req, res) =>
+                [chain, resource, _, property, _, operation, _, fmt] = req.params
+                @log.debug gLineInfo('GET v1 API'),
+                    {chain:chain, resource:resource, property:property, operation:operation, fmt:fmt}
+                if chain is 'resolver'
+                    # TODO do resolver stuff
+                    res.end()
+                else
+                    return @notFound(res,400,'No Blockchain Found',chain,->) if not (resolver=@dnschain.chains[chain])
+                    return @notFound(res,400,'No Resource Found',resource,->) if not (resolveResource=resolver.resources[resource])
+                    resolveResource.call resolver, property, operation, fmt, req.query, @postResolveCallback(res, property,->)
 
+            # support old api
+            router.get '/*', @callback.bind(@)
+
+            app.use(router)
```

Comment:
Look at the last row in the last table here:

http://expressjs.com/4x/api.html#app.use

I'm thinking what you have above should be done differently.

Try creating two routers, not one.

The first router should be mounted at `"/v1"`, and the second one should be mounted at `"/"` (I think).